backend/pah_fm/permissions.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import permissions
+from fleet_management.models import User
+
+class UserHasGroupPermission(permissions.BasePermission):
+    message = 'You are not allowed to log in because you are not associated to proper group.'
+
+    def has_permission(self, request, view):
+        try:
+            return User.objects.get(email=request.data.get('username')).groups.exists()
```

Comment:
```suggestion
            return User.objects.get(username=request.data.get('username')).groups.exists()
```
`username` is the primary key in this model.